release-notes.txt

```diff
@@ -1,10 +1,38 @@
+OpenMDAO Version 1.6.4 Alpha Release Notes
+May 10, 2016
+
+Features:
+* User can get more compact check_partials.
+* User can call alloc_jacobian() to pre-allocate jacobian with correct sizes for everything.
+* Setup messages are now all delayed and given at the end of setup.
+* Component solve failure is now reported to driver.
+* Systems (Groups & Components) will set a _jacobian_changed flag, so that solver can know if they need to do some extra work.
+* Pyoptsparse Driver will catch AnalysisError and pass fail-flag to pyopt-sparse.
+* Added error msg to top of run and run_once in case force_fd is changed from its setting pre-setup.
+* User can call list_params method on a group to get a list of all the parameters that don't have any src at that level of the hierarchy.
+* Add a test support utility function to check for the presence of fd_options['force_fd']=True in a group.
+* Improve direct solver performance by manually assembling the jacobian.
+* We now raise an error if more than one objective is added to a driver that does not support multiple objectives.
+* Added ability for user to get some basic profiling information.
+* Many improvements to the model structure viewer.
+* Many documentation fixes and improvements.
+
```

Comment:
Add:
"User can specify a scaler when adding an output or state. That number is internally used to scale the variable in the unknowns vector."
"User can specify a resid_scaler when adding a state. That number is internally used to scale the resids vector."